server/game/cards/09.3-JfS/UjikTactics.js

```diff
@@ -0,0 +1,22 @@
+const DrawCard = require('../../drawcard.js');
+const { Durations } = require('../../Constants');
+const AbilityDsl = require('../../abilitydsl');
+
+class UjikTactics extends DrawCard {
+    setupCardAbilities() {
+        this.action({
+            title: 'Give each non-unique character +1 military during this conflict',
+            condition: () => this.game.isDuringConflict(),
+            gameAction: AbilityDsl.actions.cardLastingEffect(context => ({
+                target: context.player.cardsInPlay.filter(card => card.inConflict && !card.isUnique()),
+                effect: AbilityDsl.effects.modifyMilitarySkill(1),
+                duration: Durations.UntilEndOfConflict
+            })),
+            effect: 'give all non-unique character they control +1 military skill'
```

Comment:
If you make this 
`effect: 'give all non-unique characters they control +1{1}'
effectArgs: ['military']`
Then it will use the military skill symbol.